Sources/PackageModel/Manifest.swift

```diff
@@ -15,12 +15,14 @@ import Utility
 public enum ManifestVersion: String, Codable {
     case v4 = "4"
     case v4_2 = "4_2"
+    case v5 = "5"
```

Comment:
I am actually currently working on adding support for manifest v5. I should be done soon, let's wait until I merge my changes as there are some subtle things to take care of when adding a new manifest version.

Reply:
@aciidb0mb3r thanks, now that your change has landed, I removed this